dash_uploader/configure_upload.py

```diff
@@ -2,7 +2,7 @@
 import os
 
 from dash import __version__ as dashversion
-import dash_html_components as html
+from dash import html

```

Comment:
As the dash version 2.0.0 was just released (2021-09-04), I expect 99% of user will be getting import errors with this. The old way still works. Seems that this import does not exist in the dev branch anymore, though.